lib/mpl_toolkits/axes_grid/axes_rgb.py

```diff
@@ -1,7 +1,6 @@
-from mpl_toolkits.axes_grid1.axes_rgb import (
-    make_rgb_axes, imshow_rgb, RGBAxesBase)
+from mpl_toolkits.axes_grid1.axes_rgb import make_rgb_axes, imshow_rgb, RGBAxes
 from mpl_toolkits.axisartist.axislines import Axes
 
 
-class RGBAxes(RGBAxesBase):
+class RGBAxes(RGBAxes):
```

Comment:
??? auto-replaced?

Reply:
as above

Reply:
Oh, I see. I would rather use a different name to make it less confuing. Either
~~~
from mpl_toolkits.axes_grid1.axes_rgb import RGBAxes as _RGBAxes
class RGBAxes(_RGBAxes):
~~~
or 
~~~
import mpl_toolkits.axes_grid1.axes_rgb
class RGBAxes(mpl_toolkits.axes_grid1.axes_rgb.RGBAxes):
~~~

Reply:
Sure.  I changed axisartist as you suggested and made axes_grid just star-import from axisartist, as 1) they're exactly the same, 2) axes_grid (not axes_grid1) is deprecated anyways, and 3) star-importing from axisartist is likewise used for other parts of axes_grid, e.g. angle_helper.